engine/render/src/test/test_render.cpp

```diff
@@ -1199,6 +1220,18 @@ TEST(Constants, NamedConstantsArray)
     }
     ASSERT_VEC4(original_values[2], values[6]);
 
+    ////////////////////////////////////////////////////////////
+    // Test: Test iteration of the names constants
+    ////////////////////////////////////////////////////////////
+    IterConstantContext iter_ctx;
```

Comment:
Test-wise isn't it a bit risky to only test iteration over one value? Depending on the underlying data structure it feels like it could leave certain bugs undiscovered.